### PR TITLE
Add sybil attack glossary item [Fixes #12096]

### LINKED ISSUE #12096: Sybil attack as a glossary item
### Checklist

- [X] This is a term not already found in the glossary (for similar terms, please consider the benefits of a new term vs updating an existing term)
- [X] This term/definition is not a product advertisement or contain other promotional content
- [X] This term/definition is directly relevant to Ethereum

### Term name

Sybil attacks

### Term definition

Individual humans trick a system into thinking they are multiple people to increase their influence.

### Sources, if any (please do not submit copywrited content without appropriate approval)

https://ethereum.org/decentralized-identity#sybil-protection

### Ethereum.org links

https://ethereum.org/developers/docs/consensus-mechanisms#sybil-chain

### Additional context

This term is mentioned in multiple places on the website - it would be nice to make it a glossary term and link it. https://ethereum.org/developers/docs/consensus-mechanisms here mentioned at the beginning of the document, later in the document explained what it is. It would be nice to know what it is when first encountered. 

### Would you like to work on this issue?

- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by aslikaya**
## Description

[Fixes #12096]

I searched through the website and added this as a tooltip on the first occurrence of this term on pages.

### CODE REVIEW

**Review by wackerow — CHANGES_REQUESTED** [review]
Overall looks good @aslikaya! But ["Sybil" should remain capitalized](https://en.wikipedia.org/wiki/Sybil_attack) throughout here

**Comment by aslikaya** [comment]
I thought they were mistakes😅 changing back now

File: public/content/developers/docs/consensus-mechanisms/index.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Watch more on the different types of consensus mechanisms used on Ethereum:
...
 
-**Sybil resistance** measures how a protocol fares against a [Sybil attack](https://wikipedia.org/wiki/Sybil_attack). Sybil attacks are when one user or group pretends to be many users. Resistance to this type of attack is essential for a decentralized blockchain and enables miners and validators to be rewarded equally based on resources put in. Proof-of-work and proof-of-stake protect against this by making users expend a lot of energy or put up a lot of collateral. These protections are an economic deterrent to Sybil attacks.
+**Sybil resistance** measures how a protocol fares against a Sybil attack. Resistance to this type of attack is essential for a decentralized blockchain and enables miners and validators to be rewarded equally based on resources put in. Proof-of-work and proof-of-stake protect against this by making users expend a lot of energy or put up a lot of collateral. These protections are an economic deterrent to Sybil attacks.
```
[the remark this answers is not in the document]
Comment (aslikaya) [reply public/content/developers/docs/consensus-mechanisms/index.md:71]: on the upper side of the page there is a tooltip explanation on the first occurrence of the term, shall I still put it here?

[the remark this answers is not in the document]
Comment (wackerow) [reply public/content/developers/docs/consensus-mechanisms/index.md:71]: @aslikaya Ah right, just once per page is fine... I see why you removed this now. 👍

[the remark this answers is not in the document]
Comment (aslikaya) [reply public/content/developers/docs/consensus-mechanisms/index.md:71]: @wackerow I can still add here too btw, no problem at all

[the remark this answers is not in the document]
Comment (wackerow) [reply public/content/developers/docs/consensus-mechanisms/index.md:71]: All good 👍, better to not overload with links

**Review by wackerow — APPROVED** [review]
Looks great! Thanks @aslikaya!

**Pull request merged**